WordPress/src/main/java/org/wordpress/android/ui/prefs/accountsettings/AccountSettingsViewModel.kt

```diff
@@ -149,7 +149,10 @@ class AccountSettingsViewModel @Inject constructor(
 
     fun onPasswordChanged(newPassword: String) {
         showChangePasswordDialog(true)
-        onAccountSettingsChange { pushAccountSettingsUseCase.updatePassword(newPassword) }
+        val onSuccess = {
+            updateToastMessageUiState(resourceProvider.getString(R.string.change_password_confirmation))
+        }
+        onAccountSettingsChange(onSuccess = onSuccess) { pushAccountSettingsUseCase.updatePassword(newPassword) }
```

Comment:
@SowmiaS Suggestion 💡 : Can we change the function from `onAccountSettingsChange` to `onAccountSettingsChanged`. That feels much better. WDYT?. 🤔 . Also, `SiteViewModel` feels like a view model rather than a UI Model. Can we rename it to SiteUiModel?.